conda-forge.yml

```
@@ -18,5 +22,6 @@ github:
   tooling_branch_name: main
 provider:
   linux_aarch64: default
+  linux_ppc64le: default
   win: azure
-test: native_and_emulated
+test: native
```

Try cross-compilation testing again

```suggestion
test: native_and_emulated
```